styles.css

```diff
@@ -559,3 +597,612 @@ label {
+    -moz-transform: rotate(-90deg);
+    -ms-transform: rotate(-90deg);
+    -o-transform: rotate(-90deg);
+    transform: rotate(-90deg);
+  }
+
+  #item2 {
+    grid-column-start: 3;
+    grid-column-end: 6;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    position: relative;
+  }
+
+  #item2 .infoBox {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    margin: 0;
+    width: 100%;
+    height: 63%;
+  }
+
+  #item2 .projInfo {
+    margin-top: 60px;
+    margin-left: 60px;
+  }
+
+  #item2 .projInfo h3 {
+    margin: 0;
+    margin-bottom: 34px;
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 80%;
+  }
+
+  #item2 .projInfo ul {
+    margin: 0;
+    width: 65%;
+    margin-bottom: 60px;
+  }
+
+  #item2 .projInfo ul li {
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+  }
+
+  .btnProj {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 27px;
+    color: #1a2236;
+    background-color: white;
+  }
+
+  .btnIcon {
+    background-color: black;
+    color: white;
+    filter: invert(100%);
+  }
+
+  #item2 .projInfo .lang1 {
+    margin-left: 0;
+  }
+
+  #item3 {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    display: grid;
+    align-items: flex-end;
+  }
+
+  #item3 .infoBox {
+    display: none;
+  }
+
+  #item4 {
+    grid-column-start: 2;
+    grid-column-end: 5;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    background-image: url(.png/w-image-geometry_1.png);
+    background-size: contain;
+    background-position: bottom;
+    background-repeat: no-repeat;
+  }
+
+  #item5 {
+    grid-column-start: 5;
+    grid-column-end: 6;
+    grid-row-start: 2;
+    grid-row-end: 3;
+  }
+
+  #item5 .infoBox {
+    display: none;
+  }
+
+  #item6 {
+    grid-column-start: 1;
+    grid-column-end: 4;
+    grid-row-start: 3;
+    grid-row-end: 4;
+  }
+
+  #item6 .infoBox {
+    display: none;
+  }
+
+  #item7 {
+    grid-column-start: 4;
+    grid-column-end: 6;
+    grid-row-start: 3;
+    grid-row-end: 4;
+    background-image: url(.png/w-image-geometry_2.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: bottom, right;
+  }
+
+  #item8 {
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 4;
+    grid-row-end: 5;
+    background-image: url(.png/w-image-geometry_3.png);
+    background-repeat: no-repeat;
+  }
+
+  #item9 {
+    grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 4;
+    grid-row-end: 5;
+    background-image: url(.png/w-image-geometry_4.png);
+    background-repeat: no-repeat;
+    background-position: 100% 0%;
+  }
+
+  #item10 {
+    grid-column-start: 4;
+    grid-column-end: 6;
+    grid-row-start: 4;
+    grid-row-end: 5;
+  }
+
+  #item10 .infoBox {
+    display: none;
+  }
+
+  #itemFlow {
+    position: absolute;
+    right: 0;
+    top: 620px;
+    display: grid;
+  }
```

Comment:
- [ ] Your project is matchng the figma design very well, only the second project is not aligned properly, it should share the first one's row but is bigger in height your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206276506-e1d27d67-5af9-4a35-bcac-73b892e415f5.png)

figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206276713-75011cfa-5a34-4b18-b1e6-62c2356f2b60.png)

Reply:
I have corrected the size of the item, if it`s still wrong please give a code hint as I don`t know more than what I already did, it looks correct when the browser width is 1440 as Figma width.